MApp.js

```diff
@@ -0,0 +1,110 @@
+document.addEventListener('DOMContentLoaded', () => {
+   fetch('https://ghibliapi.herokuapp.com/films') //we want the data here before user interacton
+   .then(response => response.json())
+
+   .then(dataObj => {
+      fetchMovie(dataObj)
+
+      // fetchMovie(response)
+   })
+
+   // .catch(err =>{
+   //    console.log("Nah Dawg", err)
+   // })
+
+   eventListeners()
+})
+
+var iDontBelong = [null] //API response 
```

Comment:
If you wanted to use this variable just to save id of movies, you could just leave it empty like "[]" this. Also another instance of confusing variable name that could be better named. Variable shouldn't sound like a song name unless it's necessary 👍